playbooks/common/openshift-cluster/upgrades/pre/verify_memory_and_diskspace.yml

```diff
@@ -0,0 +1,12 @@
+- name: Verify Host Requirements
+  hosts: OSEv3
```

Comment:
This should be `oo_all_hosts`.  `OSEv3` should not be called in `common` playbooks.